.gitpod.yml

```
@@ -0,0 +1,10 @@
+tasks:
+  - init: make deps generate && make build && docker-compose up -d redis mysql mailhog saml_idp && ./build/fleet prepare db --dev
+    command: ./build/fleet serve --dev
```

Just noticed that if you turn off TLS, Gitpod exposes things properly (with a valid cert and everything!)
```suggestion
    command: ./build/fleet serve --dev --server_tls=false
```

---

Hmm, yeah, I see what you mean. Maybe we would want to do the thing like we do in preview where we run the two servers (one with TLS, one without).

I'm fine with merging and iterating.